packages/flutter/lib/src/widgets/scroll_metrics.dart

```diff
@@ -36,7 +36,7 @@ abstract class ScrollMetrics {
   ///
   /// This is useful if this object is mutable, but you want to get a snapshot
   /// of the current state.
-  ScrollMetrics cloneMetrics() => new FixedScrollMetrics.clone(this);
+  ScrollMetrics toMetrics() => new FixedScrollMetrics.clone(this);
```

Comment:
In particular, PageMetrics is definitely not the same type as FixedScrollMetrics. I mean, they share a common interface type, but they're in every other way quite different.